kahuna/public/js/components/gr-tooltip/gr-tooltip.js

```diff
@@ -3,23 +3,26 @@ import 'titip';
 
 export const tooltip = angular.module('grTooltip', []);
 
-tooltip.directive('grTooltip', ['onValChange', function (onValChange) {
-    return {
-        restrict: 'A',
-        link: function ($scope, element, attrs) {
-            const position = attrs.grTooltipPosition || 'bottom';
+tooltip.directive('grTooltip', [
+    'onValChange',
+    '$interpolate',
+    function (onValChange, $interpolate) {
+        return {
+            restrict: 'A',
+            link: function ($scope, element, attrs) {
+                const position = attrs.grTooltipPosition || 'bottom';
+                const toolTipText = $interpolate(attrs.grTooltip);
```

Comment:
Why are we using interpolate?

Not saying its wrong, I've just never used it/don't really know what it does 😄